AIDevGallery/Samples/Open Source Models/Language Models/SemanticKernelChat.xaml.cs

```diff
@@ -0,0 +1,267 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using AIDevGallery.Models;
+using AIDevGallery.Samples.Attributes;
+using AIDevGallery.Samples.SharedCode;
+using Microsoft.Extensions.AI;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.ChatCompletion;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Input;
+using Microsoft.UI.Xaml.Media;
+using System;
+using System.Collections.ObjectModel;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace AIDevGallery.Samples.OpenSourceModels.LanguageModels;
+
+[GallerySample(
+    Name = "Semantic Kernel Chat",
+    Model1Types = [ModelType.LanguageModels],
+    Id = "0d20e41d-d2d4-4e49-b55a-2a38a02db482",
+    Icon = "\uE8D4",
+    Scenario = ScenarioType.TextSemanticKernelChat,
+    NugetPackageReferences = [
+        "CommunityToolkit.Mvvm",
+        "Microsoft.ML.OnnxRuntimeGenAI.DirectML",
+        "Microsoft.SemanticKernel.Core"
+    ],
+    SharedCode = [
+        SharedCodeEnum.GenAIModel,
+        SharedCodeEnum.ChatTemplateSelector,
+        SharedCodeEnum.Message,
+    ])]
```

Comment:
Updating to match recent changes to how language samples are added

```suggestion
    NugetPackageReferences = [
        "CommunityToolkit.Mvvm",
        "Microsoft.SemanticKernel.Core"
    ],
    SharedCode = [
        SharedCodeEnum.ChatTemplateSelector,
        SharedCodeEnum.Message,
    ])]
```